packaging/deb/dotnet-sharedframework-debian_config.json

```diff
@@ -32,5 +32,9 @@
         "%HOSTFXR_DEBIAN_PACKAGE_NAME%" : {},
         "libssl1.0.0" : {},
         "libcurl3" : {}
-    }
+    },
+
+    "debian_ignored_dependencies" : [
+        "liblldb-3.6"
+    ]
```

Comment:
Why is this change only necessary in the `dotnet-sharedframework-debian_config.json` file, and not also for the other .json files in this directory, for the other hostfxr and sharedhost packages?